packages/communication-react/review/communication-react.api.md

```diff
@@ -1127,9 +1127,9 @@ export type ParticipantsRemovedListener = (event: {
 
 // @public (undocumented)
 export interface PlaceholderProps {
-    displayName?: string;
-    noVideoAvailableAriaLabel?: string;
-    userId?: string;
+    displayName?: string | undefined;
```

Comment:
I'm confused. `displayName?` already means it's optional, no?

Reply:
Its optional in the SDK but I am starting to question if it should be optional in the UI components. This would be a good question to raise if there is a use case for having no name?

Silly idea. Perhaps we use the acs id if the no display name is set? The only way for someone to go out of their way to have no name is to explicitly apply "" to the displayName prop?

Reply:
Good point @prprabhu-ms 
Will update the code

@alkwa-msft Contoso might want a minimalistic UI and not want to display the name. I think it's better to have a choice than not have one at all.

Reply:
With that being said, I have removed the option from `VideoGallery` as it is already becoming a god object. Will think of a better way to compose a video gallery.

Reply:
This comment is resolved but still shows `?: .. | undefined` - is this change pending a push?

Reply:
Forgot to rebuild the package.

Reply:
Does this resolve your concern @JamesBurnside

Reply:
Not quite, the question here is why have both `?` and ` | undefined`
Is it not sufficient to keep this as `displayName?: string;` and not have the `| undefined`

Reply:
@JamesBurnside I think you are referring to the `| undefined` check there.
I was getting type error without using undefined inside VideoTile.

Reply:
Looks like I was getting weird type errors due to a faulty vs code.
I have reverted unnecessary type modifications.